README.md

```diff
@@ -265,6 +265,61 @@ The static site that gets built is pushed to the `documentation` branch of this
 We host the site on netlify. On master branch builds (see `.github/workflows/documentation.yml`), we push the built docs to
 the `documentation` branch. Netlify automatically re-deploys the docs pages whenever there is a change to that branch.
 
+## Releases
+### Release Timeline
+**For Rasa Open Source, we usually commit to time-based releases, specifically on a monthly basis.** 
+This means that we commit beforehand to releasing a specific version of Rasa Open Source on a specific day, 
+and we cannot be 100% sure what will go in a release, because certain features may not be ready. 
+
+At the beginning of each quarter, the Rasa team will review the scheduled release dates for all products and make sure 
+they work for the projected work we have planned for the quarter, as well as work well across products.
+
+**Once the dates are settled upon, we update the respective milestones.**
+
+### Cutting a Major / Minor release
+#### A week before release day
+
+1. **Make sure the [milestone](https://github.com/RasaHQ/rasa/milestones) already exists and is scheduled for the 
+correct date**
+2. **Take a look at the issues & PRs that are in the milestone**: does it look about right for the release highlights 
+we are planning to ship? Does it look like anything is missing? (don't worry about being aware of every PR that should 
+be in, but it's useful to take a moment to evaluate what's assigned to the milestone)
+3. **Post a message on the engineering Slack channel**, letting the team know you'll be the one cutting the upcoming 
+release, as well as:
+    1. Providing the link to the appropriate milestone
+    2. Reminding everyone to go over their issues and PRs and please assign them to the milestone
+    3. Clarifying the scheduled date for the release
+
+#### Release day! 🤘
+
+1. **At the start of the day, post a small message on slack announcing release day!** Communicate you'll be handling 
```

Comment:
Again: Isn't that too late? Engineers should be aware of who is cutting the release way before the release date in my opinion.

Reply:
Well just above (a week before) there was already a part of the instructions that asked for this same communication (so supposedly people already know a week in advance, but are merely reminded on the day as well) -- I personally tend to shoot on the side of over-communicating, but again if it adds confusion I can remove this second instance 😅